### PR TITLE
Topic tuneup - sales folder, pt 2 - GH 725

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) is part 2 of the clean up effort for #725 and addresses the topics in the sales folder. Upon walkthrough of the procedures, some screens were also added or updated as needed. Also some reorganization to streamline and simplify the excessive nesting. Other basic tuneup items:

- Image captions using sentence case
- Headings using sentence case and not infinitives
- Consistency in numbered step structure and style
- Edits for clarity
- Remove "cartoon balloons" from screen caps

Related PR: #751 

Internal-only staging build at: https://docs.magedevteam.com/24/user-guide/v2.3/

Note: This will be applied first to the 2.3.x production docs and then forward ported

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/v2.3/sales/credit-company-activity.html
https://docs.magento.com/user-guide/v2.3/sales/credit-company-reimburse.html
https://docs.magento.com/user-guide/v2.3/sales/credit-company.html
https://docs.magento.com/user-guide/v2.3/sales/credit-configure.html
https://docs.magento.com/user-guide/v2.3/sales/credit-memo-create.html
https://docs.magento.com/user-guide/v2.3/sales/credit-memo-print.html
https://docs.magento.com/user-guide/v2.3/sales/credit-memos.html
https://docs.magento.com/user-guide/v2.3/sales/dispatches.html
https://docs.magento.com/user-guide/v2.3/sales/fraud-protection-signifyd.html
https://docs.magento.com/user-guide/v2.3/sales/gift-options-tax.html
https://docs.magento.com/user-guide/v2.3/sales/gift-options.html
https://docs.magento.com/user-guide/v2.3/sales/gift-wrap.html
https://docs.magento.com/user-guide/v2.3/sales/invoice-create.html
https://docs.magento.com/user-guide/v2.3/sales/invoice-print-multiple.html (renamed and combined with other print content)
https://docs.magento.com/user-guide/v2.3/sales/invoices.html
https://docs.magento.com/user-guide/v2.3/sales/quote-example.html
https://docs.magento.com/user-guide/v2.3/sales/quote-price-negotiation.html
https://docs.magento.com/user-guide/v2.3/sales/quote-request.html
https://docs.magento.com/user-guide/v2.3/sales/quote-workflow.html
https://docs.magento.com/user-guide/v2.3/sales/quotes.html
https://docs.magento.com/user-guide/v2.3/sales/refunds-customer-account.html
https://docs.magento.com/user-guide/v2.3/sales/return-product-workflow.html (removed and content combined with overview topic)
https://docs.magento.com/user-guide/v2.3/sales/returns.html
https://docs.magento.com/user-guide/v2.3/sales/rma-configure.html
https://docs.magento.com/user-guide/v2.3/sales/rma-customer-experience.html
https://docs.magento.com/user-guide/v2.3/sales/rma-lifecycle.html
https://docs.magento.com/user-guide/v2.3/sales/rma-workflow.html (removed and content combined with overview topic)
https://docs.magento.com/user-guide/v2.3/sales/sales-menu.html
https://docs.magento.com/user-guide/v2.3/sales/shipment-cancel.html
https://docs.magento.com/user-guide/v2.3/sales/shipments-create.html
https://docs.magento.com/user-guide/v2.3/sales/shipments.html
https://docs.magento.com/user-guide/v2.3/sales/shopping-assistance.html
https://docs.magento.com/user-guide/v2.3/sales/shopping-assisted-cart-manage.html
https://docs.magento.com/user-guide/v2.3/sales/store-credit-using.html
https://docs.magento.com/user-guide/v2.3/sales/store-credit-workflow.html (removed and content combined with overview topic)
https://docs.magento.com/user-guide/v2.3/sales/store-credit.html
https://docs.magento.com/user-guide/v2.3/sales/terms-and-conditions.html
https://docs.magento.com/user-guide/v2.3/sales/transactions.html

### CODE REVIEW

File: src/sales/shopping-assisted-cart-manage.md [hunk 1]
```diff
@@ -3,169 +3,163 @@ ee_only: true
...
-        ![]({% link images/images-ee/customer-account-manage-cart-order-configurable-product.png %}){: .zoom}
-        _Configure_
+1. In the grid, click **Configure** next to the product name.
 
-        In the _Associated Products_ section, choose each product option to describe the item to be ordered. Then, enter the **Quantity**.
+   ![]({% link images/images-ee/customer-account-manage-cart-order-configurable-product.png %}){: .zoom}
+   _Configure_
 
-   - Click **OK**.
+1. In the _Associated Products_ dialog, choose each product option to describe the item to be ordered, enter the **Quantity**, nd click <span class="btn">OK<span/>.
```
Comment (erikmarr) [inline src/sales/shopping-assisted-cart-manage.md:73]:
```suggestion
1. In the _Associated Products_ dialog, choose each product option to describe the item to be ordered, enter the **Quantity**, and click <span class="btn">OK<span/>.
```

**Review by erikmarr — APPROVED** [review]
Looks good. Just minor feedback.

File: src/sales/invoice-create.md [hunk 2]
```diff
@@ -4,94 +4,88 @@ title: Creating an Invoice
...
 
-![]({% link images/images/sales-invoices.png %}){: .zoom}
-_Invoices_
+{:.bs-callout-info}
+Do not select the _Not Capture_ option unless you are certain that you are going to capture the payment through Magento at a later date. You cannot create a credit memo until the payment has been captured using the Capture button.
 
 ## Invoice an order
 
-1. On the _Admin_ sidebar, click **Sales**.
+You must generate an invoice for an order before you can [print it]. To view or print the PDF, first download and install a PDF reader such as [Adobe Acrobat Reader][1].
```
Comment (leslietilling) [inline src/sales/invoice-create.md:25]:
```suggestion
You must generate an invoice for an order before you can print it. To view or print the PDF, you must have a PDF reader. You can download [Adobe Reader][1] at no charge.
```

**Comment by leslietilling** [comment]
running tests

**Pull request merged**